test/Tests/src/Network/Http_Spec.enso

```diff
@@ -293,28 +293,14 @@ spec =
             test_file = enso_project.data / "sample.txt"
             response = Data.post url_post (Request_Body.Binary test_file)
             response.at "headers" . at "Content-Type" . should_equal "text/plain"
-            response.at "data" . should_equal 'Cupcake ipsum dolor sit amet. Caramels tootsie roll cake ice cream. Carrot cake apple pie gingerbread chocolate cake pudding tart souffl\u00E9 jelly beans gummies.'
+            expected_text = test_file.read_text
+            response . at "data" . should_equal expected_text
 
         Test.specify "Can perform a binary File POST" <|
             test_file = enso_project.data / "sample.png"
             response = Data.post url_post (Request_Body.Binary test_file)
-            expected_response = Json.parse <| '''
-                {
-                  "headers": {
-                    "User-Agent": "Java-http-client/17.0.7",
-                    "Content-Type": "text/plain",
-                    "Content-Length": "26905"
-                  },
-                  "origin": "127.0.0.1",
-                  "url": "",
-                  "method": "POST",
-                  "form": null,
-                  "files": null,
-                  "data": "\uFFFDPNG",
-                  "json": null,
-                  "args": {}
-                }
-            response . should_equal expected_response
+            response.at "headers" . at "Content-Type" . should_equal "text/plain"
```

Comment:
Why is the content type `text/plain` here?

Reply:
We don't detect the content type of files, but we should.

Reply:
Defaulting to application/octet-stream would be good here though.

Reply:
> Defaulting to application/octet-stream would be good here though.

Yep, as long as we cannot detect a more precise content type, I think octet stream is a better default. PNG has nothing to do with txt

Reply:
Done.